src/ontology/mondo-ingest.Makefile

```diff
@@ -592,11 +595,51 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 ##################################
 ## Externally managed content ####
 ##################################
-# General
-update-externally-managed-content: external-content-nord external-content-nando external-content-ordo external-content-omim
 
 EXTERNAL_CONTENT_DIR=external
 
+EXTERNAL_FILES = \
+	mondo-clingen \
+	mondo-efo \
+	mondo-medgen \
+	mondo-omim-genes \
+	mondo-otar-subset \
+	nando-mappings \
+	gard \
+	nord \
+	ordo-subsets
+
+# The following target simulates the EMC pipeline. It runs it in exactly the same way
+# as it would be run in the context of the Mondo repo. This way, we can test Mondo 
+# in exactly the same state as it would look like when the EMC pipeline was run properly.
+# In particular, this allows us to use QC tests that are reliant on the various pre- and 
+# postprocessing steps needed, like Animal QC (which relies on the rare subset pipeline,
+# which is SPARQL postprocessing based).
+
+$(TMPDIR)/mondo-with-simulated-emc-run.owl: $(TMPDIR)/mondo_repo_built $(foreach n,$(EXTERNAL_FILES), $(EXTERNAL_CONTENT_DIR)/$(n).robot.owl)
+	mkdir -p $(TMPDIR)/mondo/src/ontology/tmp/external/
+	$(foreach n,$(EXTERNAL_FILES), cp -f $(EXTERNAL_CONTENT_DIR)/$(n).robot.owl $(TMPDIR)/mondo/src/ontology/tmp/external/processed-$(n).robot.owl;) \
+	cd $(TMPDIR)/mondo/src/ontology && \
+	touch mondo-edit.owl && \
+	cp mondo-edit.owl mondo-edit.owl.bak && \
+	cp mondo-edit.obo mondo-edit.obo.bak && \
+	make update-external-content-incl-rare -B MIR=false IMP=false MIR=false DOWNLOAD_EXTERNAL=false &&\
```

Comment:
## Bug: `DOWNLOAD_EXTERNAL=false`
[Trish commented](https://github.com/monarch-initiative/mondo-ingest/pull/628#issuecomment-2334371732) that  `sh run.sh make update-externally-managed-content -B`, which currently includes the recent simulation goal that Nico added, is erroring: 

```
tmp/external/processed-efo-proxy-merges.robot.owl (No such file or directory)
```

The problem involves running some `make` goals in a clone of `mondo` within `mondo-ingest`'s `tmp/` dir. There is an option `DOWNLOAD_EXTERNAL` (`true` by default in the `mondo.Makefile`), which downloads files from `mondo-ingest`. However, during this simulation goal, it passes `DOWNLOAD_EXTERNAL=false`. I think that Nico added that to avoid a kind of circular dependency, as it actually downloads these files from the `mondo-ingest` repo. However, if it does not download these files, then they will not exist, causing this error.

Fixed by:
- #672 

Though I don't know if there's a good reason why Nico passed `DOWNLOAD_EXTERNAL=false`. Maybe there's a different solution he would prefer.